vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -90,7 +90,7 @@ function Disable-WindowsUpdates {
 
 function Get-ContainerImages {
     if ($containerRuntime -eq 'containerd') {
-        Write-Log "Pulling images for windows server 2019 with containerd"
+        Write-Log "Pulling images for windows server 2019/2022 with containerd"
```

Comment:
Can we separate the log for 2022 from that for 2019?

Reply:
Added variable $windowsSKU into the log. The variable $windowsSKU will be "2019-containerd", "2022-containerd", ...